### PR TITLE
created report new issue

### PR DESCRIPTION
**Pull request by diderson**
#### What's this PR do?
create a report issue form the following has been done:
1) Create a event folder reportissue in evently > reportandfix
2) Trigger this event folder from evently > reportandfix > _init > selectors (use id as selector)
3) Create a view issueCategories with reduce function which should emit issue categories.
4) Create async.js, in event folder reportissue and get the data required for categories using the view.
5) Create data.js, in event folder reportissue and prepare the data you got from async
6) Create the mustache.html in event folder reportissue to display the form.
7) on submit should save the issue in db. And have to trigger to event issuedetail with issue _id as params.

#### Where should the reviewer start?
the review should start on event reportissue

#### How should this be manually tested?

http://devtest.hsh-global.com/fixmystreet-diderson/_design/reportandfix/index.html

#### What are the relevant tickets?
 #10
#### Questions:
- Is there a blog post? N/A
- Does the knowledge base need an update? N/A
- Does this add new (Python) dependencies which need to be added to chef? N/A

### CODE REVIEW

File: evently/reportandfix/_init/after.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+function () {
+	$(this).pathbinder('storeNewIssue','/saveNewIssue')
+}
```
Comment (shyams) [inline evently/reportandfix/_init/after.js]: @diderson use tabs 2 here

File: evently/reportandfix/reportissue/selectors/#imageUpload/change.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+function loadImage() {
```
Comment (shyams) [inline evently/reportandfix/reportissue/selectors/#imageUpload/change.js]: @diderson why you using a function name of main function

File: views/issueCategories/map.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+function(doc) {
+  if (doc.type && doc.type === "category") {
+    emit(doc.name, doc);
```
Comment (shyams) [inline views/issueCategories/map.js]: @diderson do you need doc as value? You can give null here.

File: evently/reportandfix/reportissue/selectors/#imageUpload/change.js (hunk 2 above)
Comment (diderson) [inline evently/reportandfix/reportissue/selectors/#imageUpload/change.js]: @Shyam this is a new approach I think I refer to what have been done on fixmystreet A please check registering new user process.

File: evently/reportandfix/reportissue/async.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+function(cb, e, params) {
+  var url = '_view/issueCategories?include_docs=true';
```
Comment (shyams) [inline evently/reportandfix/reportissue/async.js]: @diderson you using again docs here. You getting doc twice. As a value in _view and here.

**Pull request merged**